text/0052-tx-summary-digest.md

```diff
@@ -0,0 +1,253 @@
+- Feature Name: tx_summary_digest
+- Start Date: 2022-10-08
+- MCIP PR: [mobilecoinfoundation/mcips#0052](https://github.com/mobilecoinfoundation/mcips/pull/0052)
+- Tracking Issue: [mobilecoinfoundation/mobilecoin#2686](https://github.com/mobilecoinfoundation/mobilecoin/issues/2686)
+
+# Summary
+[summary]: #summary
+
+Introduce an object called `TxSummary` which is significantly smaller than a `Tx`,
+but which with small additional data can be used to verify the values of inputs
+and outputs, and the destinations of the outputs, when a device is asked to sign
+a `RingMLSAG` for a new transaction.
+
+Where previously the `extended_message_digest` is signed by Ring MLSAGs
+in a transaction, now a new digest formed using that digest and the `TxSummary` will
+be signed instead.
+
+# Motivation
+[motivation]: #motivation
+
+Hardware wallets such as Ledger have an expectation that when the user signs a
+transaction, the device can display information about how much money will be sent
+to what addresses as a result of this transaction, and that the device can verify
+that this information is correct.
+
+However, with the status quo, the only way that this can be accomplished is to put
+the entire `Tx` on the device so that it can verify the `extended_message_digest`,
+because the `extended_message_digest` is the only thing that connects the `MLSAG`
+that the device is signing to the rest of the `Tx`.
+
+This increases the amount of data that has to be sent to the device from a few kb
+to `> 100kb` in the worst case. This is because Merkle proofs are each e.g. 40 bytes
```

Comment:
Nit: Capitalize KB
```suggestion
This increases the amount of data that has to be sent to the device from a few KB
to >100 KB in the worst case. This is because Merkle proofs are each ~40 bytes
```